### PR TITLE
Add brochure

### PR DESCRIPTION
**Pull request by Anbranin**
Right now the brochure, as a PDF, is emailed to passengers manually by dispatchers after registration. Eventually I plan to have a callback on Passenger that automatically emails them a link to the brochure after creation. But for now this is just putting the page up. I plan to have the ST admin folks look it over and see if they like the page. If they do I'll move on to the next stage in the process.

### CODE REVIEW

File: app/views/passengers/_policies.html.haml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    service animals.
+  %li
+    Our lifts are an extremely safe way to board or exit a van, but proper lift
+    use requires cooperation between the driver and the passenger. Once the lift
+    is ready for you to board, you should position yourself on the lift with
+    your back facing the van. Please lock your wheels or turn the power off to
+    prevent any movement of the chair while the lift is in motion.
+  %li 
+    As a passenger in SpecTrans vans you are required, by Massachusetts State
+    Law: MGL Part 1, Title XIV, Chapter 90, Section 13A, to have your seat belt
```
[the remark this answers is not in the document]
Comment (Anbranin) [reply app/views/passengers/_policies.html.haml:27]: YEP.

File: app/views/passengers/_policies.html.haml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    fastened at all times while the vehicle is in motion.
+  %li 
+    Rides from an off-campus location to another off-campus locations are
+    limited to two per week, after 5PM.
```
[the remark this answers is not in the document]
Comment (Anbranin) [reply app/views/passengers/_policies.html.haml:31]: This wording doesn't come from me but I bet they wouldn't mind if I made it clearer.

**Comment by werebus** [comment]
BTW, If you want to run my grammar/style suggestions by the ST staff, this repo is public...

**Comment by Anbranin** [comment]
> BTW, If you want to run my grammar/style suggestions by the ST staff, this repo is public...

I figure they'll want to thoroughly look over the brochure as it will look to everyone, since they may, when they read through the whole thing, have more wording changes than just these to make.

Since we're not deploying this to the public yet (although the public could technically see it, they won't unless we send them the link) I figured it'd be fine to just deploy, tell them what I changed, and they can then look it over any time they like. It's low impact and if it has the wrong wording it doesn't matter yet. Alternatively we could do the whole ngrok thing but timing makes that a more annoying choice.

**Comment by werebus** [comment]
That sounds fine too, but a PR review is literally a way to display proposed edits (to text that presumably someone in their department wrote). I just thougt that that might be more polite than, "Here's the new brochure. We _did_ change some (unspecified) things."

**Comment by Anbranin** [comment]
Good point, I'll send them the PR link as well as tell them they'll be able to view it in situ.

**Comment by Anbranin** [comment]
I forgot about this PR, but now I believe I've fixed everything. ⚡

**Pull request merged**